docs/specs/moniker.yml

```diff
@@ -1028,3 +1028,41 @@ inputs:
 outputs:
   .errors.log: |
     ["error","json-syntax-error","Unexpected end when deserializing array.","monikerDefinition.json",2]
+---
+# Invalid moniker range in config
+inputs:
+  docfx.yml: |
+    monikerRange:
+      'docs/v1/**': '>= netcore-2.0'
+    monikerDefinition: monikerDefinition.json
+  docs/v1/a.md:
+  monikerDefinition.json: |
+    {}
+outputs:
+  .errors.log: |
+    ["error","moniker-range-invalid","Invalid moniker range: '>= netcore-2.0': Moniker `netcore-2.0` is not defined"]
```

Comment:
we don't have a file for this error?

Reply:
Added

---
In reply to: [286751764](https://github.com/dotnet/docfx/pull/4600#discussion_r286751764) [](ancestors = 286751764)